packages/react-components/src/CopyButton.tsx

```diff
@@ -15,27 +15,27 @@ interface Props {
   children?: React.ReactNode;
   className?: string;
   icon?: IconName;
-  isAddress?: boolean;
+  type?: string;
   isMnemonic?: boolean;
   value: string;
 }
 
 const NOOP = () => undefined;
 
-function CopyButton ({ children, className, icon = 'copy', isAddress = false, isMnemonic = false, value }: Props): React.ReactElement<Props> {
+function CopyButton ({ children, className, icon = 'copy', type = 'other', value }: Props): React.ReactElement<Props> {
```

Comment:
```suggestion
function CopyButton ({ children, className, icon = 'copy', type, value }: Props): React.ReactElement<Props> {
```

(Just removed it here so the string is translatable in full)